lib/src/components/NxGlobalSidebar/NxGlobalSidebar.scss

```diff
@@ -46,6 +51,7 @@ $nx-global-sidebar-separator-color: $nx-grey-900;
   display: flex;
   height: $nx-global-sidebar-logo-height;
   margin-bottom: $nx-spacing-xl;
+  padding: 0 $nx-spacing-l;
```

Comment:
We probably want a variable for this side spacing, similar to how we've got `$nx-modal-padding` in `NxModal.scss`

Reply:
> We probably want a variable for this side spacing, similar to how we've got `$nx-modal-padding` in `NxModal.scss`

Ok: https://github.com/sonatype/sonatype-react-shared-components/pull/387/commits/b92f7b4328c1463abfe3627d566376ae75ecc837